extensions/sql-bindings/src/extension.ts

```diff
@@ -34,6 +34,9 @@ export async function activate(context: vscode.ExtensionContext): Promise<IExten
 		},
 		getAzureFunctions: async (filePath: string): Promise<GetAzureFunctionsResult> => {
 			return getAzureFunctions(filePath);
+		},
+		addNugetReferenceToProjectFile: async (projectFile: string): Promise<void> => {
```

Comment:
Since this specification adds the `SQL` to the name of the function name since it adds the SQL NuGet package to the project file, I would specifically add that to the name since we are exposing it now.

https://github.com/microsoft/azuredatastudio/blob/f55f7a7667958fffca133822dc7d19e36ed631cc/extensions/sql-bindings/src/common/azureFunctionsUtils.ts#L240